airflow-core/src/airflow/ui/openapi-gen/queries/customInfiniteQueries.ts

```diff
@@ -0,0 +1,38 @@
+import { InfiniteData, useInfiniteQuery, UseInfiniteQueryOptions } from "@tanstack/react-query";
+
+import { DagService } from "openapi/requests/services.gen";
+
+import * as Common from "./common";
+
+export const useDagServiceGetDagTagsInfinite = <
```

Comment:
Since this is custom let's use a simpler name like `useDagTagsInfinite`

Reply:
+1 for this